### PR TITLE
Data availability api docs

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: en/developers-guide/data-availability-layer-api.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+# Pull Oracle
+
+Through the pull-based oracle, price data can be updated on-chain on demand by the user. Off-chain price data, which is updated at subsecond intervals, can be accessed and submitted to the chain on demand.
+
+## API Endpoint
+
+- Testnet (REST): https://dal.baobab.orakl.network/api/v1
+- Mainnet (REST): https://dal.cypress.orakl.network/api/v1
+
+The API requires the `X-API-Key` header for both REST API calls and WebSocket connections. Please [contact us](mailto:business@orakl.network) to receive a valid API key.
```
Comment (Intizar-T) [inline en/developers-guide/data-availability-layer-api.md:14]: I wonder if the biz team would want to include some pricing plans in the docs...

File: en/developers-guide/data-availability-layer-api.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+{
+"method":"SUBSCRIBE",
+"params":
+[
+"submission@BTC-USDT",
+"submission@ETH-USDT",
+...
+]
+}
+```
```
Comment (Intizar-T) [inline en/developers-guide/data-availability-layer-api.md:333]: minor suggestion, how about indenting to make it look prettier/formatted?

**Review by Intizar-T — APPROVED** [review]
lgtm!

File: en/developers-guide/data-availability-layer-api.md (hunk 1 above)
Comment (nick-bisonai) [reply en/developers-guide/data-availability-layer-api.md:14]: it is dev docs so I think it should be okay 🤔

**Pull request merged**